main.py

```diff
@@ -0,0 +1,116 @@
+from Board import Board
+from Snake import Snake
+from Ladder import Ladder
+
+import random
+
+def roll_die():
+    return random.randint(1, 6)
+
+
+def create_board():
+    size = int(input())
+    no_of_snakes = int(input())
+    snakes = []
+    for i in range (0, no_of_snakes):
+        start, end = input().split(' ')
+        snake = Snake(int(start), int(end))
+        snakes.append(snake)
+    no_of_ladders = int(input())
+    ladders = []
+    for i in range (0, no_of_ladders):
+        start, end = input().split(' ')
+        ladder = Ladder(int(start), int(end))
+        ladders.append(ladder)
+
+    no_of_players = int(input())
+    players = []
+    for i in range (0, no_of_players):
+        players.append(input().split(' '))
+
```

Comment:
Don't leave unnecessary blank lines. If possible read about PEP8 and enable that in your IDE.